bitnami/seaweedfs/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: seaweedfs
 sources:
 - https://github.com/bitnami/containers/tree/main/bitnami/seaweedfs
-version: 0.3.0
+version: 0.3.1
```

```suggestion
version: 0.4.0
```

You are creating a new variable so bumping the chart to a new minor version

---

I'll apply the changes to release this today